stella_transforms.f90

```diff
@@ -547,6 +548,10 @@ subroutine transform_kalpha2alpha(gkalph, galph)
 
    end subroutine transform_kalpha2alpha
 
+   ! input galph array is real and contains values on the padded alpha grid
+   ! gkalph is output array; it contains the Fourier coefficients of galph
+   ! for positive ky values only (reality can be used to obtain the negative ky coefs)
+   ! the highest 1/3 of the ky modes from the FFT have been discarded to avoid de-aliasing
```

Comment:
```suggestion
   !> input galph array is real and contains values on the padded alpha grid
   !> gkalph is output array; it contains the Fourier coefficients of galph
   !> for positive ky values only (reality can be used to obtain the negative ky coefs)
   !> the highest 1/3 of the ky modes from the FFT have been discarded to avoid de-aliasing
```

Reply:
I'm going to go ahead and add a few of these suggestions to batch.  Michael, you may want to review the ones that are leftover.